core/quorum_initiator.go

```diff
@@ -440,9 +449,15 @@ func (c *Core) finishConsensus(id string, qt int, p *ipfsport.Peer, status bool,
 	}
 }
 
-func (c *Core) connectQuorum(cr *ConensusRequest, addr string, qt int) {
+func (c *Core) connectQuorum(cr *ConensusRequest, addr string, qt int, pledgetokens float64) {
```

Comment:
`pledgetokens` is not used anywhere . Also for later , Please use camelCase while naming variables

Reply:
Removed the variable pledgetokens